internal/mill/exec_test.go

```diff
@@ -68,6 +68,24 @@ func TestRun(t *testing.T) {
 	}
 }
 
+func TestDryRun(t *testing.T) {
+	f := newPetFixtureDryRun(t)
+	defer f.tearDown()
+
+	petsitter, stdout := f.petsitter, f.stdout
+	file := filepath.Join(f.dir, "Petsfile")
+	ioutil.WriteFile(file, []byte(`run("echo meow")`), os.FileMode(0777))
+	err := petsitter.ExecFile(file)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	out := stdout.String()
```

Comment:
maybe also check that the 'dry run' output is in stderr?

Reply:
Ahhh. I can see that it is when I run the test but I'm not doing a good job getting to it. :grimacing: